mod-1/bloco-06/6.1/exercicios/form.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Form Currículo</title>
+</head>
+<body>
+  <form action="/" id="resume" method="POST" enctype="multipart/form-data">
+    <fieldset id="personal-info" name="personal-info">
+      <legend>Informações Pessoais</legend>
+      <input type="text" id="pi-name" name="fullname" placeholder="Nome" maxlength="40" required>
+      <input type="email" name="email" id="pi-email" placeholder="E-mail" maxlength="50" required>
+      <input type="text" id="pi-cpf" name="cpf" placeholder="CPF" minlength="11" maxlength="11" required>
+
+      <input type="text" id="pi-city" name="city" placeholder="Cidade" maxlength="28" required>
+      <select name="state" id="pi-state" required></select>
+
+      <fieldset id="home-type">
+        <legend>Tipo de Moradia</legend>
+        <input type="radio" name="home-type" id="pi-ht-house" value="house" required>
+        <label for="pi-ht-house">Casa</label>
+        <input type="radio" name="home-type" id="pi-ht-appartment" value="appartment" required>
+        <label for="pi-ht-appartment">Apartamento</label>
+      </fieldset>
+    </fieldset>
+
+    <fieldset name="previous-job" id="previous-job">
+      <legend>Último Emprego</legend>
+      <textarea name="summary" id="pj-summary" cols="70" rows="10" placeholder="Fale um pouco sobre você." maxlength="1000" required></textarea>
+
+      <input type="text" id="pj-role" name="role" placeholder="Cargo" maxlength="40" required>
+
+      <textarea name="description" id="pj-descritpion" cols=70" rows="10" placeholder="Descreva seu cargo." maxlength="500" required></textarea>
```

Comment:
Fala Inácio! Aqui nessa linha você esqueceu de abrir as aspas em `cols`.

Reply:
É verdade! Muito obrigado por indicar